apps/app-get-started/src/App.vue

```diff
@@ -0,0 +1,240 @@
+                  />
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M15 12a3 3 0 11-6 0 3 3 0 016 0z"
+                />
+                </svg>
+                <dt>Experiments</dt>
+                <dd>Information about how the samples were processed</dd>
+              </div>
+              <div>
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="none"
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                  stroke="currentColor"
+                  stroke-width="2"
+                  aria-hidden="true"
+                >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M9 12h6m-6 4h6m2 5H7a2 2 0 01-2-2V5a2 2 0 012-2h5.586a1 1 0 01.707.293l5.414 5.414a1 1 0 01.293.707V19a2 2 0 01-2 2z"
+                />
+                </svg>
+                <dt>Files</dt>
+                <dd>The location of the files that were generated</dd>
+              </div>
+            </dl>
+          </div>
+        </div>
+      </section>
+      <section aria-labelledby="solverd-releases-title" id="solverd-releases">
+        <h2 id="solverd-releases-title">Releases</h2>
+        <p>Click the name of the release below to view all tables available in RD3. Follow the links to view the data. If you would like to view a certain patch, click one of links below and select the "patch" filter.</p>
+        <div class="col-md-8 m-auto">
+          <Accordion
+            v-for="pkg in packages"
+            :key="pkg.id"
+            :id="pkg.id"
+            :title="pkg.label"
+            :links="pkg.links"
+          />
+        </div>
+      </section>
+    </div>
+  </div>
+</template>
+
+<script>
+// import PageComponent from '@molgenis/molgenis-ui-context/src/components/PageComponent.vue'
+import Accordion from './components/Accordion.vue'
+import axios from 'axios'
+
+export default {
+  data () {
+    return {
+      packages: {}
+    }
+  },
+  components: {
+    // PageComponent,
+    Accordion
+  },
+  mounted () {
+    axios.all([
+      this.getEmxPackages()
+    ]).then(() => {
+      this.getEmxEntities()
+    })
+  },
+  methods: {
+    async getEmxPackages () {
+      const response = await axios.get('/api/v2/sys_md_Package?attrs=id,label&q=parent==rd3')
+      response.data.items.forEach((d, i) => {
+        if (!(d.id in this.packages)) {
+          this.$set(this.packages, d.id, {})
+        }
+        this.packages[d.id].id = d.id
+        this.packages[d.id].label = d.label
```

Comment:
Can there be data with the same id? if not the if is not needed. Also this could be done in one assignment:
this.$set(this.packages, d.id, {id: d.id, label: d.label}